### ISSUE TITLE
Is urban hydrology also covered?

### ISSUE THREAD

**Issue by dleutnant** [ropensci/hydrology#25]
First of all, thank you for promoting hydrology related packages through this Task View! This will be a great resource for hydrologists starting with R to do their analysis. In this respect, I am wondering whether the **urban** hydrology field is also covered? For example, I am working a lot with the well known Stormwater Management Model [SWMM](https://www.epa.gov/water-research/storm-water-management-model-swmm) by the US EPA, for which I created the R package [`{swmmr}`](https://cran.r-project.org/package=swmmr) to interface with model results and the model itself. Do you think this fits to the hydrology CTV?

**Comment by dleutnant** [comment]
Great! Thank you!

**Comment by boshek** [comment]
Agreed! Thanks for the suggestion. Going to re-open until it is added.

**Comment by dleutnant** [comment]
I just stumbled across the TV (which is a really fantastic compilation, thank you!) and I think that the swmmr package would fit more into the section "Process-based modeling (scripts for preparing inputs/outputs and running process-based models)" because it exactly does i) preparing inputs/outputs and ii) running the model.

**Comment by boshek** [comment]
Will do!
